shell/browser/api/electron_api_session.cc

```diff
@@ -48,6 +48,7 @@
 #include "net/base/completion_repeating_callback.h"
 #include "net/base/load_flags.h"
 #include "net/base/network_anonymization_key.h"
+#include "net/base/registry_controlled_domains/registry_controlled_domain.h"
```

Comment:
needed for `net::registry_controlled_domains::INCLUDE_PRIVATE_REGISTRIES`

:+1: